src/EditAnnotations.cpp

```diff
@@ -1535,26 +1548,40 @@ Annotation* EngineMupdfCreateAnnotation(EngineBase* engine, AnnotationType typ,
 
     pdf_update_annot(ctx, annot);
     auto res = MakeAnnotationPdf(epdf, annot, pageNo);
+
+    std::optional <PdfColor> col;

```

Comment:
We don't use std:: if we can avoid it. It causes terrible bloat of executables.

Set to `ColorUnset` instead.